### PR TITLE
Social icon updates

### PR DESCRIPTION
**Pull request by bdashrad**
- Add LinkedIn
- Fix github svg spacing
- Update colors to ones from brandcolors.net

### CODE REVIEW

File: layouts/partials/social-follow.html [hunk 1]
```diff
@@ -20,6 +20,11 @@
     {{ partial "svg/youtube.svg" (dict "size" $icon_size) }}
   </a>
 {{ end }}
+{{ with .Site.Params.linkedin }}
+  <a href="{{ . }}" class="link-transition linkedin link dib z-999 pt3 pt0-l mr3" title="LinkedIn link">
```
Comment (adambkaplan) [inline layouts/partials/social-follow.html]: I've personally found the `mr3` spacing is a bit wide.

**Comment by budparr** [comment]
Hi guys. Looks like two competing PRs? I'm going to try to pull these in, but FYI I'm working Ananke 2.0, which will break these changes slightly.

**Comment by adambkaplan** [comment]
Yep - competing. @bdashrad and I seem to have wanted to add GH and LinkedIn at the same time 😺

**Comment by budparr** [comment]
Is the substantive difference the `mr3` `mr2`thing?

File: layouts/partials/social-follow.html (hunk 1 above)
Comment (bdashrad) [reply layouts/partials/social-follow.html]: good catch. i was just about to push a fix but it was just merged.

**Pull request merged**